project/PekkoBuild.scala

```diff
@@ -308,8 +308,8 @@ object PekkoBuild {
         UsefulTask("sortImports", "Sort the imports"),
         UsefulTask("mimaReportBinaryIssues ", "Check binary issues"),
         UsefulTask("validatePullRequest ", "Validate pull request"),
-        UsefulTask("docs/paradox", "Build documentation"),
-        UsefulTask("docs/paradoxBrowse", "Browse the generated documentation"),
+        UsefulTask("docs/paradox", "Build documentation(takes ~10+ minutes)"),
```

Comment:
Sorry one more minor thing, there should be a space between `Build documentation` and `(takes ~10+ minutes)` i.e. it should be `Build documentation (takes ~10+ minutes)`